nginx/nginx.conf

```diff
@@ -1,19 +1,14 @@
-upstream roboprop {
-    server web:8000;
-}
-
 server {
-
     listen 80;
 
     location / {
-        proxy_pass http://roboprop;
+        proxy_pass http://web:8000;
```

Comment:
hoping this makes static images a bit happier on the k8s side of things

Reply:
Is `web` a known name in k8s?

In k8s we run nginx in the same network namespace as django, so i believe localhost would be the correct name?

Reply:
yes a good point, and the comment was incorrect.
This was to get running correctly in docker, and the nginx configuration is set separately in k8s.
So there it would be either localhost (as it is now), or possibly roboprop